.github/workflows/ci_edalize.yml

```
@@ -0,0 +1,25 @@
+name: build-openlane-sky130
+on: [push]
+
+jobs:
+  build-picorv32:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          path: picorv32
+      - name: Checkout pdk
+        uses: actions/checkout@v2
+        with:
+          repository: olofk/pdklite
+          path: pdklite
+      - run: echo "PDK_ROOT=$GITHUB_WORKSPACE/pdklite" >> $GITHUB_ENV
+      - run: echo "EDALIZE_LAUNCHER=${GITHUB_WORKSPACE}/openlane_runner.py" >> $GITHUB_ENV
+      - run: pip3 install --user -e "git+https://github.com/olofk/edalize.git#egg=edalize"
+      - run: pip3 install fusesoc
+      - run: docker pull efabless/openlane:v0.12
+      - run: wget https://raw.githubusercontent.com/olofk/subservient/main/openlane_runner.py
```

The olofk/subservient has switched away from using the `openlane_runner.py`.  Is this change sufficient to pass the CI or should it be updated to the newer Edalize docker setup?
```suggestion
      - run: wget https://raw.githubusercontent.com/klasnordmark/openlane-examples/main/openlane_runner.py
```